### ISSUE TITLE
Getting "blocked by: maximum OCU capacity reached" errors when harvesting

### ISSUE THREAD

**Issue by tloubrieu-jpl** [NASA-PDS/registry#340]
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

When I did harvest a bundle, I received the following logs.  Bug raised by @schole-ds.

<img width="951" alt="image" src="https://github.com/user-attachments/assets/ebef1798-e23f-4952-94ac-d1e887eeade0">

### 🕵️ Expected behavior

I expected harvrst to succeed.

### 📚 Version of Software Used

harvest 4.0.3

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

Need to check what the impact of the OCU limit is on disk space.

**Comment by tloubrieu-jpl** [comment]
Might be a duplicate.

**Comment by jordanpadams** [comment]
@tloubrieu-jpl just wanted to doublecheck on this. it does not appear to be an issue right now?

**Comment by tloubrieu-jpl** [comment]
Yes, let's close this ticket. Harvest can now wait when the maximum capacity of opensearch is reached.

**Comment by c-suh** [comment]
@tloubrieu-jpl please provide test data for me to verify Harvest's behavior when the maximum capacity of OpenSearch is reached. Ty!

**Comment by c-suh** [comment]
i&t.skip label added because it is not worth reproducing a heavy load in production from registry.
